scrape.py

```diff
@@ -283,6 +297,11 @@ def main():
 
     for group in files:
         for build in group['builds']:
+            # Just use the specified data when the scraper cannot be used
+            if build.get('scraper') == 'noscraper':
+                build['link'] = build['url']
+                continue
```

Comment:
Instead of using `continue`, the following `if build.get('scraper')` could be an `elsif`; one less line and easier to follow.